### PR TITLE
docs: ARM Linux might need `cpan`

### PR DESCRIPTION
**Pull request by osalbahr**
The tldr is that the `ruby` package might be too barebones in some distribution such as Fedora. I have not tested on Ubuntu.

- See https://github.com/orgs/Homebrew/discussions/5688#discussioncomment-11056174

Update: the same error does not occur in `ubuntu:rolling`

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

**Comment by osalbahr** [comment]
Closing. This isn't needed unless the error also gets reported in another distro imho.

File: docs/Homebrew-on-Linux.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ To install build tools, paste at a terminal prompt:
 
 Homebrew can run on 32-bit ARM (Raspberry Pi and others) and 64-bit ARM (AArch64), but as they lack binary packages (bottles) they are unsupported. Pull requests are welcome to improve the experience on ARM platforms.
 
-You may need to install your own Ruby using your system package manager, a PPA, or `rbenv/ruby-build` as we no longer distribute a Homebrew Portable Ruby for ARM.
+You may need to install your own Ruby using your system package manager, a PPA, or `rbenv/ruby-build` as we no longer distribute a Homebrew Portable Ruby for ARM. You might also need to install the `cpan` package.
```
Comment (MikeMcQuaid) [inline docs/Homebrew-on-Linux.md:82]: I know this was closed anyway but: I really don't want the ARM documentation here to grow over time. It's unsupported: that's all we really need to say here.
